lsbi/network.py

```diff
@@ -112,13 +132,6 @@ def fit(self, X, y, **kwargs):
             mean_loss = torch.mean(torch.tensor(epoch_loss)).item()
             print(f"Epoch {epoch+1}/{num_epochs}, Loss: {mean_loss}")
 
-        # Evaluate the model
```

Comment:
What are these lines doing (no tests break, but they look like they could be important)

Reply:
So I thought these were here for a reason and upon checking I think without switching to eval(), batch norm does the wrong thing at inference time, I've added the switch back, but the whole model can just be turned now it is wrapped.

I thought about adding a test for this but the result is not an error just misleading numerical values so it seems a bit overkill to test